docs/content/en/docs/installation.md

```diff
@@ -32,18 +32,4 @@ To make use of the package, you need a running instance of GoodData. If you do n
 
 ### Versioning
 
-The Python SDK is versioned. The version you should be using depends on the type of GoodData distribution you are using.
-
-#### GoodData Cloud
-
-We recommend you always work with the newest (supported) Python available.
```

Comment:
Note that we should mention that Python SDK works with GoodData Cloud, so users know that they cannot use it with GoodData Platform.